.devcontainer/images/Dockerfile

```
@@ -30,10 +30,13 @@ USER root
 # =============================================================================
 # Dynamic Binary Tools (rtk) — kubectl/helm/bazelisk/yq are in base
 # =============================================================================
-RUN RTK_LATEST=$(curl -fsSL ${GITHUB_TOKEN:+-H "Authorization: token ${GITHUB_TOKEN}"} "https://api.github.com/repos/rtk-ai/rtk/releases/latest" | jq -r '.tag_name') && \
-    echo "Installing rtk ${RTK_LATEST}..." && \
+# Pinned to a known-good rtk release. See CLAUDE.md §2.0 bump policy.
+# MUST stay in sync with RTK_PINNED_VERSION in .devcontainer/install.sh
+# (drift caught by tests/scripts/install-sh-rtk.bats test 9).
+ARG RTK_VERSION=v0.38.0
+RUN echo "Installing rtk ${RTK_VERSION}..." && \
```

Hadolint (2.14.0)

[warning] Set the SHELL option -o pipefail before RUN with a pipe in it. If you are using /bin/sh in an alpine image or if your shell is symlinked to busybox then consider explicitly setting your SHELL to /bin/ash, or disable this check

(DL4006)